system/src/ledger/ledger_manager.cpp

```diff
@@ -259,9 +260,12 @@ int LedgerManager::init() {
     if (!buf) {
         return SYSTEM_ERROR_NO_MEMORY;
     }
+    auto fsInstance = filesystem_get_instance(FILESYSTEM_INSTANCE_DEFAULT, nullptr);
+    assert(fsInstance);
+    FsLock fs(fsInstance);
```

Comment:
Why does this one need to change but not the one in `LedgerManager::getLedgerNames`?

Reply:
The manager is expected to be initialized in `getLedgerNames()` and `instance()` will take care of that.

Reply:
The main change in `init()` is that it now calls `filesystem_mount()` and the latter expects a `filesystem_t` instance, not `lfs_t`.